### ISSUE TITLE
How to organize my folders when using KERAS-CV for Object Detection

### ISSUE THREAD

**Issue by silvada95** [keras-team/keras-cv#1972]
Hi, 

I am trying to use Keras-CV Object Detection model, but I am a bit confused about how my folders should be organized for using the dataloading functions.

For example, when using YOLOv8 on PyTorch, the organization generally is:

Dataset Folder
                        Images  (jpg, png, and so on)
                                     Train 
                                     Val 
                        Labels (txt)
                                  Train 
                                  Val

Does this structure work easily with Keras-CV, or is it necessary to create custom functions before loading it into Keras detection pipelines?

**Comment by jbischof** [comment]
KerasCV is totally agnostic to how you organize your data. However, if you want to use Keras helper functions to help create a data generator there are some constraints. See https://keras.io/api/data_loading/
